writer_test.go

```diff
@@ -1225,6 +1225,27 @@ func ExampleMediaPlaylist_Segments_scte35_oatcls() {
 	// media2.ts
 }
 
+func ExampleMediaPlaylist_Segments_scte35_cue() {
```

Comment:
I can't see any uses of this function, not any tests added that test the writing functionality (ideally via setting the new tags programmatically). Could you please add at least one?

Reply:
What this function is testing is that it is able to build the proper segments from a manifest, and then checks that the resulting manifests match the original manifest. But yes, I will add another test that builds the manifest programmatically.

Reply:
Removed this test and created a new one that tests the same functionality in: https://github.com/zencoder/m3u8/pull/8/commits/c953678b1aa9fccf3c8ab6bb7b540872f75ba4c0